### PR TITLE
Allow for updating ebury associations and association accounts

### PR DESCRIPTION
**Pull request by mebowler**

### CODE REVIEW

File: form3/resource_ebury_gateway_association_account.go [hunk 1]
```diff
@@ -219,3 +221,103 @@ func createEburyAssociationAccountFromResourceData(d *schema.ResourceData) (*mod
...
+
+func createEburyUpdateAssociationAccountFromResourceData(d *schema.ResourceData) (*models.EburyAssociationAccountUpdate, error) {
+	associationAccount := models.EburyAssociationAccountUpdate{
+		Type: string(models.ResourceTypeEburyAssociationAccounts),
+		Attributes: &models.EburyAssociationAccountPatchAttributes{
+			AccountWith: &models.EburyAssociationAccountPatchAttributesAccountWith{},
+		},
+	}
+
+	// TODO this is not documented as `association_account_id` in the swagger...
```
Comment (aleksandra-form3) [inline form3/resource_ebury_gateway_association_account.go:266]: Leftover?

File: swagger.yaml [hunk 2]
```diff
@@ -9826,6 +9972,70 @@ definitions:
...
+        type: string
+        format: uuid
+        x-nullable: false
+      type:
+        type: string
+        x-nullable: false
+      version:
+        type: integer
+        minimum: 0
+        x-nullable: true
```
[the remark this answers is not in the document]
Comment (mebowler) [reply swagger.yaml]: Correct, this was just copied over from other resources. This is just a definition to create the client, the behaviour is implemented in ebury-gateway and versions are correctly checked there

[the remark this answers is not in the document]
Comment (mebowler) [reply swagger.yaml]: updated to reflect the correct behaviour

**Review by aleksandra-form3 — APPROVED** [review]

**Comment by mebowler** [comment]
Not currently required due to recent changes with the International project

**Pull request closed without merging**